DeDRM_plugin/adobekey.py

```diff
@@ -129,7 +149,10 @@ class ADEPTError(Exception):
         c_long, c_ulong
 
     from ctypes.wintypes import LPVOID, DWORD, BOOL
-    import _winreg as winreg
```

Comment:
There is `six` syntax for this one.

Reply:
Consequence of the intention to vendor six. Changed now.